build.sbt

```diff
@@ -0,0 +1,45 @@
+
+name := "salesforce-message-handler"
+
+organization := "com.gu"
+
+description:= "handle outbound messages from salesforce to update zuora and indentity"
+
+version := "1.0"
+
+scalaVersion := "2.11.11"
```

Comment:
I started using 2.12.2 for my new lambda, although it relies on the libraries you need being available